fastmri/models/varnet.py

```diff
@@ -184,8 +196,21 @@ def forward(self, masked_kspace: torch.Tensor, mask: torch.Tensor) -> torch.Tens
         num_low_freqs = torch.max(
             2 * torch.min(left, right), torch.ones_like(left)
         )  # force a symmetric center unless 1
+
+        if isinstance(self.num_sens_lines, int):  # Use prespecified number instead
```

Comment:
nit: prefer this pattern, might work if it's a non-Python int.
```suggestion
        if self.num_sens_lines is not None:  # Use prespecified number instead
```